workflows/ci-build-yocto.yaml: add file

Paullgk opened the pull request:
This commit adds a workflow file with the objective to build a SEAPATH Yocto image, inside a GitHub action.
This first version tests the installation of dependencies and the sources intialization needed to build a SEAPATH Yocto image.

eroussy commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
```
This file should not be there

Paullgk committed:
workflows/ci-build-yocto.yaml: add file

This commit adds a workflow file with the objective to build a SEAPATH
Yocto image, inside a GitHub action.
This first version tests the installation of dependencies and the
sources intialization needed to build a SEAPATH Yocto image.

Signed-off-by: Paul Le Guen de Kerneizon <paul.leguendekerneizon@savoirfairelinux.com>
